src/people.rs

```diff
@@ -789,6 +861,61 @@ pub trait ContextPeopleExt {
     /// `context.query_people(((Age, 30), (Gender, Female)))`.
     fn query_people<T: Query>(&self, q: T) -> Vec<PersonId>;
     fn match_person<T: Query>(&self, person_id: PersonId, q: T) -> bool;
+    fn get_counts<T: Tabulator, F>(&self, tabulator: &T, print_fn: F)
```

Comment:
Architecture:
I wouldn't call this `get_counts()` but rather `tabulate_person_properties()` or the like.